### PR TITLE
Changing the Order Workflow

### PR DESCRIPTION
**Pull request by shabananavas**
Issue 104.

### CODE REVIEW

File: v2/orders/order-processing.rst [hunk 1]
```diff
@@ -4,3 +4,39 @@ Order Processing
 Order processing is part of the order refresh process. This is run when
 on draft orders to ensure that it has up to date adjustments and that
 its order items are up to date.
+
+In Commerce Core, the order process moves throug, h the following cycle: Cart -> Validation -> Fulfillment -> Completed. It starts with the order being in the shopping cart, which is the Cart state, then, once the order is placed, it is put in the Validation state. Once we're ready to ship the goods, the order is moved to the Fulfillment state. And finally, once it leaves our store, the order is officially Completed.
```
Comment (joshmiller83) [inline v2/orders/order-processing.rst]: typo in "through" of the first sentence. Let's take the time to use lucidchart.com and create a nicer graph.

File: v2/orders/order-processing.rst [hunk 2]
```diff
@@ -4,3 +4,39 @@ Order Processing
...
+
+Now, that we know the process, let's take a look at how we can create orders on behalf of customers and move them along the order life cycle.
+
+Creating an Order
+-----------------
+Site administrators can create orders on behalf of their customers by going to ``/admin/commerce/orders/add``. From here, we can either create a new order for an existing customer (chosen from the autocomplete search box). Or, we can create a new customer on the fly by providing just an email address.
+
+.. image:: https://user-images.githubusercontent.com/11439056/28088259-c5191542-66a2-11e7-9e3f-016ed0e5f60f.png
```
Comment (joshmiller83) [inline v2/orders/order-processing.rst]: Images need to be committed

File: v2/orders/order-processing.rst [hunk 3]
```diff
@@ -4,3 +4,39 @@ Order Processing
...
+
+.. image:: https://user-images.githubusercontent.com/11439056/28088341-009a8696-66a3-11e7-8725-8a427b8787a6.png
+
+As we move further down, we'll see that there is an "Adjustments" section. This where we can add promotions, add a shipping amount, add tax, as well as, any custom amount to the order total.
+ 
+ And finally, we can apply coupons to the order.
+ 
+.. image:: https://user-images.githubusercontent.com/11439056/28088650-f8d08a4a-66a3-11e7-9992-0b14612e4ee5.png
+
+If we already have promotions running, this will automatically be reflected in the item pricing and the overall order total.
```
Comment (joshmiller83) [inline v2/orders/order-processing.rst]: Sidenote link to the promotions documentation?

File: v2/orders/order-processing.rst [hunk 4]
```diff
@@ -4,3 +4,39 @@ Order Processing
...
+
+Now that we've added all the order details, let's save the order. We also have the option of saving this new order to our cart. This will automatically add the products in this order to our shopping cart so we can complete the checkout by going to ``/cart``. 
+
+.. image:: https://user-images.githubusercontent.com/11439056/28088833-9d349bb2-66a4-11e7-8ead-b2b8ee0c356d.png
+
+Now, click to view the order. Notice that a discount has been automatically applied to the order total as we had a "20% Off" promotion running store-wide. Also, notice that the order is currently in "Draft" state. 
+
+As an admin, once we've got all the order details done, our next job would be to complete payment on the order. That's where the 'Payments' tab comes in. The payments page allows us to process a Credit Card/Email Money Transfer/Bank Transfer/Cheque payment for the order using the store's preferred Payment Gateway.
```
Comment (joshmiller83) [inline v2/orders/order-processing.rst]: Needs screenshots

File: v2/orders/order-processing.rst [hunk 5]
```diff
@@ -4,3 +4,64 @@ Order Processing
 Order processing is part of the order refresh process. This is run when
 on draft orders to ensure that it has up to date adjustments and that
 its order items are up to date.
+
+In Commerce Core, the order process moves through the following cycle:
+
+.. image:: ../images/order_workflow.png
```
Comment (mglaman) [inline v2/orders/order-processing.rst]: This is not the default workflow

Comment (joshmiller83) [inline v2/orders/order-processing.rst]: Good point. I think Shabana is basing this off our urban hipster demo which is not a default workflow.

**Comment by shabananavas** [comment]
@joshmiller83 @mglaman My apologies there on the confusion. I have modified the title (@joshmiller83 please feel free to come up with something more adequate) and the intro to be more accurate.

**Comment by mglaman** [comment]
This is covering changing the workflow and managing an order. The scope is split. Is this about managing an order or how to use a validation workflow

**Comment by shabananavas** [comment]
It's about how to change a workflow to the Validation workflow and use that workflow.

**Comment by mglaman** [comment]
Got it, I think we can merge this, then merge in #108 and follow up with linking them together.

**Pull request merged**